Scripts/Modules/SnappingModule/SnappingModuleUI.cs

```diff
@@ -0,0 +1,88 @@
+﻿#if UNITY_EDITOR
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace UnityEditor.Experimental.EditorVR
+{
+	public class SnappingModuleUI : MonoBehaviour
```

Comment:
Is this the Settings UI? If so, should probably be called SnappingModuleSettingsUI.